deepmedic/routines/testing.py

```diff
@@ -404,7 +404,13 @@ def inferenceWholeVolumes(  sessionTf,
     log.print3("###########################################################################################################")
     log.print3("############################# Finished full Segmentation of " + str(validation_or_testing_str) + " subjects ##########################")
     log.print3("###########################################################################################################")
-    return 0
+
+    metrics_dict_list = []
+    for i in range(len(meanDiceCoeffs1)):
```

Comment:
If ground truth labels are not given (e.g. for testing) then Dice Coffs are undefined. Just inference is done and results are saved.

In that case, error occurs. E.g. try:
python ./deepMedicRun -model examples/configFiles/tinyCnn/model/modelConfig.cfg -test examples/configFiles/tinyCnn/test/testConfig.cfg -load some-pretrained-model

I think this can be avoided if these are moved 5 lines further up, at the end of the block:
if listOfFilepathsToGtLabelsOfEachPatient is not None and total_number_of_images>0 :

Reply:
fixed as suggested